Dockerfile

```diff
@@ -14,8 +15,8 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF505
 
 COPY ./ ./
 
-RUN curl -L -o ballot https://github.com/scality/ballot/releases/download/v0.0.0-alpha.20210121162209/ballot-20210121162209-linux-amd64 \
-    && chmod +x ballot
+ADD https://github.com/scality/ballot/releases/download/v${BALLOT_VERSION}/ballot-v${BALLOT_VERSION}-linux-amd64 /usr/src/app/ballot
```

Comment:
oooh never tried using a url for ADD before.

Reply:
stole this from @tmacro :wink:

Reply:
`ADD` is nice because even if the layer is cache it will still download and checksum the src and invalidate the cache if it changes